firebase.json

```
@@ -36,17 +36,17 @@
       { "source": "/+isolates", "destination": "/language/concurrency", "type": 301 },
       { "source": "/+lexicalscope", "destination": "/language/functions#lexical-scope", "type": 301 },
       { "source": "/+pub", "destination": "/tools/pub", "type": 301 },
-      { "source": "/2.0", "destination": "/guides/language/evolution#dart-20", "type": 301 },
+      { "source": "/2.0", "destination": "/resources/language/evolution#dart-20", "type": 301 },
```

```suggestion
      { "source": "/2.0", "destination": "/resources/language/evolution#dart-2-0", "type": 301 },
```
Other one didn't work